lib/surface/content_handler.ex

```diff
@@ -72,7 +74,7 @@ defmodule Surface.ContentHandler do
 
       # TODO [Context]: Update this to be only the appropriate assigns for context
       case args[:__context__] || assigns.__context__ do
-        [] ->
+        %{} ->
```

Comment:
```suggestion
        context when context == %{} ->
```

I believe just `%{}` will always match as that will pattern match any map

Reply:
🤦‍♂️